src/electron/build.action.sh

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+PLATFORM="${1:-}"
+RELEASE="${2:-}"
```

Comment:
We should reconcile this with the `BUILD_ENV` var we pass to webpack.
Perhaps we get rid of BUILD_ENV.

Though RELEASE needs to be renamed to BUILD_TYPE or something like that.

Reply:
Yeah, I'll call it what ever we decide in our sidebar above